hypervisor/common/hypercall.c

```diff
@@ -166,21 +166,35 @@ int32_t hcall_create_vm(struct acrn_vm *vm, uint64_t param)
 			/* Filter out the bits should not set by DM and then assign it to guest_flags */
 			vm_config->guest_flags |= (cv.vm_flag & DM_OWNED_GUEST_FLAG_MASK);
 
-			/* GUEST_FLAG_RT must be set if we have GUEST_FLAG_LAPIC_PASSTHROUGH set in guest_flags */
-			if (((vm_config->guest_flags & GUEST_FLAG_LAPIC_PASSTHROUGH) != 0U)
-				&& ((vm_config->guest_flags & GUEST_FLAG_RT) == 0U)) {
-				pr_err("Wrong guest flags 0x%lx\n", vm_config->guest_flags);
+			/* post-launched VM is allowed to assign PCPU from vm_config->pcpu_affinity_bitmap only */
```

Comment:
"assign" is less accurate. To me, this is to choose.

Reply:
Sure.